redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -37,16 +39,31 @@ internal class ViewFlexContainer(context: Context, direction: FlexDirection) {
     flexDirection = direction
   }
 
-  private val _view = HostView(context)
-  val view: View get() = _view
+  private val hostView = HostView(context)
+
+  private val scrollView = if (direction.isHorizontal) {
+    HorizontalScrollView(context).apply {
```

Comment:
Maybe we can eventually extend `HorizontalScrollView`/`VerticalScrollView` to avoid creating an extra view, but it's not necessary at the moment.